Failure with numpy 1.13

We have a failure in Travis-CI when using the latest `numpy`:

```
___________________________________ test_FUV ___________________________________
    def test_FUV():
        x = loadbunch(fname, masked=False)
        # Switch epoch from Matlab to Python
        x.t -= 366
        x.t0 -= 366
    
        for i, flag in enumerate(x.flags):
>           F, U, V = FUV(x.t, x.t0, x.lind-1, x.lat, flag)
tests/test_harmonics.py:29: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../miniconda/envs/TEST/lib/python2.7/site-packages/utide/harmonics.py:170: in FUV
    freq = linearized_freqs(tref)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tref = 711859.0
    def linearized_freqs(tref):
        astro, ader = ut_astron(tref)
        freq = const.freq.copy()
>       freq[not_shallow] = np.dot(const.doodson[not_shallow, :], ader) / 24
E       TypeError: NumPy boolean array indexing assignment requires a 0 or 1-dimensional input, input has 2 dimensions
../../../miniconda/envs/TEST/lib/python2.7/site-packages/utide/harmonics.py:29: TypeError
----------------------------- Captured stdout call -----------------------------
i: 0 ngflgs: [0 0 0 0]
i: 1 ngflgs: [0 1 0 0]
```

Full log https://travis-ci.org/ocefpaf/UTide/jobs/280403569

I'll take a look soon, we should fix that before a new release.

--- a/utide/harmonics.py
+++ b/utide/harmonics.py
@@ -26,7 +26,7 @@ kshallow = np.nonzero(~not_shallow)[0]
 def linearized_freqs(tref):
     astro, ader = ut_astron(tref)
     freq = const.freq.copy()
-    selected = np.dot(const.doodson[not_shallow, ...], ader) / 24
+    selected = np.dot(const.doodson[not_shallow, :], ader) / 24
     freq[not_shallow] = selected.squeeze()
     for i0, nshal, k in zip(ishallow, nshallow, kshallow):
         ik = i0 + np.arange(nshal)

--- a/utide/harmonics.py
+++ b/utide/harmonics.py
@@ -26,7 +26,8 @@ kshallow = np.nonzero(~not_shallow)[0]
 def linearized_freqs(tref):
     astro, ader = ut_astron(tref)
     freq = const.freq.copy()
-    freq[not_shallow] = np.dot(const.doodson[not_shallow, :], ader) / 24
+    selected = np.dot(const.doodson[not_shallow, ...], ader) / 24
+    freq[not_shallow] = selected.squeeze()
     for i0, nshal, k in zip(ishallow, nshallow, kshallow):
         ik = i0 + np.arange(nshal)
         freq[k] = (freq[shallow.iname[ik] - 1] *

